introduction/upstream-kie-community.adoc

```diff
@@ -63,6 +63,6 @@ Eder's focus is associated with tools.
 There isn't a single way to track work in community.
 While the projects were sponsored by Red Hat, JIRA was used for community and also product. 
 
-Since moving to IBM, the project work is tracked at https://github.com/orgs/IBM/projects/21/views/1[IBM's GitHub board].
+Since moving to IBM, the project work is tracked at https://github.com/orgs/apache/projects/295/views/1[IBM's GitHub board].
```

Comment:
I think as we change link href, we need to change also link text?

Reply:
+1